tests/test_api_lib_policy.py

```diff
@@ -1569,7 +1569,7 @@ def test_01_check_tokentype(self):
         # The token type SPASS is not allowed on this client, so an exception
         #  is raised.
         r = check_tokentype(req, resp)
-        jresult = json.loads(r.data)
+        jresult = r.json
```

Comment:
I am starting to doubt, if we can use ``.json``.
Since we might modify ``.data`` in decorators and like this.
Or is ``.json`` simply a reprensation of  ``.data``, even if it was modifies?

Reply:
`.json` is a (cached) JSON representation of `.data`, implemented in [`werkzeug`](https://github.com/pallets/werkzeug/blob/a41a3e06ec1403f464d23431495ba59b41484e58/src/werkzeug/wrappers/json.py#L119)